source/app/main.c

```diff
@@ -23,27 +24,51 @@ Audio loadAudio() {
     return loadAudioFromROM(audioData);
 }
 
+void seedRNGByKeyPress() {
+  /* fake seeding by just fetching numbers until key is pressed. */
+  while(1) {
+    vid_vsync();
+    SimpleRNG_rand();
+    key_poll();
+    if(key_hit(KEY_ANY)){
+      break;
+    }
+  }
+}
+
 int main()
 {
```

Comment:
Dit is een ding van mij, maar het bracht me wel aan het denken: wat vind jij fijner? Haakjes op een nieuwe regel of op dezelfde? Ik vind dat beiden voor en nadelen hebben